src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -647,6 +713,15 @@ li {
   }
 }
 
+.toggle-business-entry {
+  cursor: pointer;
+  color: $app-blue;
+  text-decoration: underline;
+  float: right;
```

Comment:
FYI, there's a Vuetify class for this.

My approach: if it's a layout thing, try to do it in the template (as it's easier to manage layouts that way imo). Otherwise declare a class.

This class contains a bunch of styling stuff so I'm totally OK with it.

Reply:
Thanks, for sure!
Will make sure to use it where it exclusive additional styling 🦖

Reply:
> Thanks, for sure! Will make sure to use it where it exclusive additional styling 🦖

Perfect. This is my new rule :)

Reply:
I like it!